stubs/psutil/psutil/__init__.pyi

```diff
@@ -189,11 +193,11 @@ def net_if_addrs() -> dict[str, list[snicaddr]]: ...
 def net_if_stats() -> dict[str, snicstats]: ...
 
 if sys.platform == "linux":
-    def sensors_temperatures(fahrenheit: bool = ...): ...
-    def sensors_fans(): ...
+    def sensors_temperatures(fahrenheit: bool = ...) -> dict[str, list[shwtemp]]: ...
+    def sensors_fans() -> dict[str, list[sfan]]: ...
 
 if sys.platform != "win32":
-    def sensors_battery(): ...
+    def sensors_battery() -> sbattery: ...
```

Comment:
This returns `None` on my desktop computer.

Reply:
Because your desktop computer does not have a battery! Awesome bug! 🔍 

It can indeed return `None`: https://github.com/giampaolo/psutil/blob/dccf8bedf34fced63f1962dd77b58f0da339759c/psutil/_pslinux.py#L1399